src/replica/HttpIngestModule.cc

```diff
@@ -684,18 +685,18 @@ json HttpIngestModule::_deleteTable() {
     }
 
     database::mysql::ConnectionHandler const h(qservMasterDbConnection("qservCssData"));
-    h.conn->executeInOwnTransaction([&databaseInfo, &table](decltype(h.conn) conn) {
+    h.conn->executeInOwnTransaction([&database, &table](decltype(h.conn) conn) {
         // Remove table entry from czar's MySQL
-        conn->execute("DROP TABLE IF EXISTS " + conn->sqlId(databaseInfo.name, table));
+        conn->execute("DROP TABLE IF EXISTS " + conn->sqlId(database.name, table.name));
         // Remove the director index (if any)
-        if (databaseInfo.isDirector(table)) {
+        if (table.isDirector) {
             conn->execute("DROP TABLE IF EXISTS " +
-                          conn->sqlId("qservMeta", ::directorIndexTable(databaseInfo.name, table)));
+                          conn->sqlId("qservMeta", ::directorIndexTable(database.name, table.name)));
         }
         // Remove the row counters table (if any)
         try {
             conn->execute("DROP TABLE IF EXISTS " +
-                          conn->sqlId("qservMeta", ::rowCountersTable(databaseInfo.name, table)));
+                          conn->sqlId("qservMeta", ::rowCountersTable(database.name, table.name)));
         } catch (invalid_argument const&) {
             // This exception may be thrown by the table name generator if
             // it couldn't build a correct name due to MySQL limitations.
```

Comment:
Is it worth logining the exceptions as warnings?

Reply:
You're right - this has to be logged as `ERROR`. I will fix it.